oeps/processes/oep-0056-architectural-process.rst

```diff
@@ -0,0 +1,218 @@
+
+This difficulty is compounded by the decentralized and global nature of the Open edX community.
+
+Atrophy of architectural principles and tech radar
+--------------------------------------------------
+
+An important component of our ability to make architectural decisions that are aligned across teams and organizations is measuring and vetting those decisions against a set of `shared architectural principles <https://openedx.atlassian.net/wiki/spaces/AC/pages/921895082/Architecture+Vision+Principles>`_ and technologies in our `Tech Radar <https://github.com/openedx/openedx-tech-radar>`_.  Today, we lack organized maintenance of our principles and radar, nor do we have an appropriate forum in which to discuss and socialize them.
+
+Missing ceremonies
+------------------
+
+As a community of companies invested in the success of the Open edX platform, we lack any sort of regular ceremonies in which we discuss the platform's architectural priorities as they relate to our desired platform and organization-specific goals.  This often leads to a situation where architectural thinking is done primarily by senior engineering leaders as they learn about their business' needs, and by engineers in the course of projects, with little structured decision making in between to help bridge the highly strategic and the tactical.  Furthermore, the conversations that do happen tend to occur in an ad hoc way - often as "hallway conversations" or one-off project syncs - with whoever can be found to offer enough input to unblock the team.
+
+Specification
+=============
+
+We believe that we can create a healthy and empowering dialogue around our architectural process and mitigate the above concerns by focusing on a re-alignment of ceremonies and clarifying the spirit and process around the creation and review of ADRs.
+
+We will adjust our current architectural process by:
+
+- Formalizing and clarifying an **Architectural Advisory Process** in which we conversationally provide input on and promptly review the status of ADRs, as well as review updates to our architectural principles and tech radar, with the goal of enabling more effective, informed decision making.
+- Improve the discoverability of stakeholders and experts via a **decision maker's checklist**.
+- Documenting as a *suggested best practice* a quarterly **technical strategic review process** where we can review our technical strategy and cross-functional requirements as high-level, guiding input to the Advisory Process, and to help teams and organizations decide where to prioritize addressing "important but not urgent" work such as architectural investments, new capabilities, addressing technical debt, and performing ongoing maintenance/upgrades.
```

Comment:
First, I feel this single sentence is way too long. Secondly I'm thrown by 

> review our technical strategy and cross-functional requirements as high-level, guiding input to the Advisory Process, and to help teams and organizations decide

They feel weird and different and I'm not sure what you're trying to say. I'm trying to reword it and I'm having difficulties. Is it something like,

> Documenting, as a *suggested best practice*, a quarterly **technical strategic review process**. This process lays out how we review our technical strategy and cross-functional requirements at a high-level and how we help teams and organizations decide where to prioritize addressing "important but not urgent" work such as architectural investments, new capabilities, addressing technical debt, and performing ongoing maintenance/upgrades. This technical strategic review process provides guiding input to the Advisory Process.

I think? Having two processes, or things called processes, feels confusing/overwhelming.

Reply:
I'm actually going to be removing the whole 'technical strategic review' aspect of this OEP.  It's ill-defined and needs to be iterated on before we figure out how to codify it, IMHO.  Meanwhile, the Advisory Process part feels pretty tight.